rest/data/interface.go

```diff
@@ -131,6 +132,9 @@ type Connector interface {
 
 	// AbortPatch aborts the patch corresponding to the input patch ID and deletes if not finalized.
 	AbortPatch(string, string) error
+	// AbortPatchesFromPullRequest aborts patches with the same PR Number,
+	// in the same repository, at the pull request's close time
+	AbortPatchesFromPullRequest(*github.PullRequestEvent) error
```

Comment:
can we make this a ref, to avoid nil pointer exceptions?

Reply:
done

Reply:
The event itself is coming from go-github, which is always a non nil pointer if it passes HMAC verification, so I don't think it would make a difference here. I could throw a quick if event == nil if it's a concern to you.

Reply:
I think I'm less worried about specific errors here (indeed there are panic recovery handlers everywhere now.) and more just the general do we want to pass pointers across interface boundaries in general. 

It's probably fine to let this slide, but I do want to think about the larger question.